docs/getting-started/setup.md

```diff
@@ -34,4 +34,27 @@ executes a simple data pipeline that downloads two files from two different webs
 When the execution is complete, you should see the two files in the *data* folder.
 Wonder what happened ? You will create the data pipeline that does just this in the first steps of this guide.
 
+## Development Environment
+For some parts of this tutorial it is beneficial to have a working development environment ready. In the following we will mainly explain how one can configure a working evironment for 
+Windows or Linux. We will focus on the community version of intellij. Please [download](https://www.jetbrains.com/idea/) the version that suits your operating system. 
+### Windows
+#### System setup 
+1. First download the Windows binaries for Hadoop [here](https://github.com/cdarlint/winutils/archive/refs/heads/master.zip)
+2. Extract the wished version to a folder (e.g. &lt prefix &gt\hadoop-&lt version &gt\bin ). For this tutorial we use the version 3.2.2.
+3. Configure the *HADOOP_HOME* environment variable to point to the folder &lt prefix &gt\hadoop-&lt version &gt\
+4. Add the *%HADOOP_HOME%\bin* to the *PATH* environment variable
+   
+#### Intellij
+1. Open the project in intellj
+2. Load the Maven project
+3. Install the scala plugin
+4. Select one of the preconfigured runConfigurations (part-x)
```

Comment:
I was able to follow this until this step.
These preconfigured runConfigurations don't work yet